pkg/domain/domain.go

```diff
@@ -2840,9 +2840,8 @@ func (do *Domain) ReleaseConnID(connID uint64) {
 }
 
 // GetAutoAnalyzeProcID returns processID for auto analyze
-// TODO: support IDs for concurrent auto-analyze
 func (do *Domain) GetAutoAnalyzeProcID() uint64 {
```

Comment:
This function is not used after this patch?

Reply:
Yes, have removed.